osidb/api_views.py

```diff
@@ -331,6 +333,18 @@ def create(self, request, *args, **kwargs):
         return response
 
 
+@extend_schema_view(
```

Comment:
I am not sure if I should add here also `@include_meta_attr_extend_schema_view` and `@include_exclude_fields_extend_schema_view`.

Reply:
The schema extension is only for the sake of documentation and it should correspond to the serializer. There is no `meta_attr` in the model so no need for that one and so the serializer correctly does not inherit `IncludeMetaAttrMixin`. Also the serializer does not inherit the `IncludeExcludeFieldsMixin` and I think that is OK for such a small model. Only if we wanted to have it everywhere for the sake of consistence. I am not sure what is the good practice here. WDYT @Elkasitu ?

Reply:
I see that even `ErratumSerializer` with almost no fields inherit `IncludeExcludeFieldsMixin` so maybe we want to have it everywhere so we are consistent.

Reply:
OK, let's wait for @Elkasitu's opinion.